<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatFrames</class>
 <widget class="QStackedWidget" name="OperatFrames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="pg_sd">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="lb_sd1">
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/target-1-2);
color: rgb(255, 128, 128);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_sd2">
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/target-2-2);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_sd3">
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/target-3-2);
color: rgb(204, 102, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_sd4">
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/target-4-2);
color: rgb(0, 204, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pg_menu">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/hh1/blue-bkg);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pb_exit">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>68</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/blue-button);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_spare1">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>68</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_record">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>68</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/record);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>RECORD</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_selfTest">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>68</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/hh1/blue-button);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>SELF
TEST</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
